bancorprotocol/carbon-app
# Display fiat on budget description

fix #1244 

![image](https://github.com/user-attachments/assets/5c58bf05-4d76-4e5f-a7e1-f9b5f8b9974b)

Test case : 
- Create an overlapping strategy with one token with fiat and one without
- Enter a budget and see description: one has fiat price, the other not
